app/Jobs/Notifications/ForumTopicReply.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace App\Jobs\Notifications;
+
+use App\Models\Forum\Post;
+use App\Models\User;
+use App\Models\UserNotificationOption;
+
+class ForumTopicReply extends BroadcastNotificationBase
+{
+    public function __construct(Post $object, User $source)
+    {
+        parent::__construct($object, $source);
```

Comment:
I wonder if having `$notifiable` and `$post` (or `$comment` etc) instead of `getNotifiable` and `$object` would be easier to read.

Something like
```php
$this->post = $post;

parent::__construct($post->topic, $source);
```